src/blockchain/contracts/lido_locator.py

```diff
@@ -24,3 +26,18 @@ def staking_router(self, block_identifier: BlockIdentifier = 'latest') -> Checks
         response = self.functions.stakingRouter().call(block_identifier=block_identifier)
         logger.info({'msg': 'Call `stakingRouter()`.', 'value': response, 'block_identifier': repr(block_identifier)})
         return response
+
+    def withdrawal_queue(self, block_identifier: BlockIdentifier = 'latest') -> ChecksumAddress:
+        response = self.functions.withdrawalQueue().call(block_identifier=block_identifier)
+        logger.info({'msg': 'Call `withdrawalQueue()`.', 'value': response, 'block_identifier': repr(block_identifier)})
+        return response
+
+    @property
+    def withdrawal_queue_contract(self) -> WithdrawalQueueContract:
```

Comment:
Let's move it to LidoContracts.
This approach is ok, let's just keep consistency. Use this one or old one

Reply:
If there would be an error on contract object initialisation then in the current code we will catch it while checking possibility of sending mellow transaction. If we catch it we will send regular transaction. If I move it to the LidoContracts and there will happen an error in initialisation of this contract it will fail to start the bot.